plugins/cache/redis.py

```diff
@@ -110,17 +234,71 @@ def __init__(self, *args, **kwargs):
         self._prefix = self.get_option('_prefix')
         self._keys_set = self.get_option('_keyset_name')
         self._sentinel_service_name = self.get_option('_sentinel_service_name')
+        self._decode_responses = bool(self.get_option('_decode_responses'))
+        self._encoding = self.get_option('_encoding')
+        self._encoding_errors = self.get_option('_encoding_errors')
+        self._retry_on_timeout = bool(self.get_option('_retry_on_timeout'))
+        self._socket_keepalive = self.get_option('_socket_keepalive')
+        self._socket_connect_timeout = int(self.get_option('_socket_connect_timeout')) \
+            if self._socket_keepalive and self.get_option('_socket_connect_timeout') else None
```

Comment:
As a personal preference for style, I do like using the `x = a if b else c` construct for smaller expressions. In this case, I think it might be more readable expanding this to a "normal" `if` block.